### PR TITLE
Feed, a very basic one.

### PR DESCRIPTION
**Pull request by uxengine**
Also cleaning up utils/format.js a bit.

screen shot 2017-11-28 at 18 37 25

### CODE REVIEW

File: pages/feed.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import withMe from '../lib/apollo/withMe'
+import withT from '../lib/withT'
+
+const FeedPage = ({ url, me, t }) => {
+  const meta = {
+    title: t('pages/feed/title')
+  }
+  return (
+    <Frame raw url={url} meta={meta}>
+      {me && me.id ? <Feed /> : <Marketing />}
```
Comment (tpreusse) [inline pages/feed.js]: Could you port [Auth/withAuthorization](https://github.com/orbiting/publikator-frontend/blob/master/components/Auth/withAuthorization.js) and use that instead? We shouldn't make pages available under multiple urls. And a person having a deep link most likely needs to log in.

**Comment by uxengine** [comment]
Done. Checking against the CF 'supporter' role for now, which may or may not change with the ongoing sell&join migration.

**Comment by tpreusse** [comment]
@uxengine `supporter` gives access to the admin tools ;) but thanks for added `withAutherization`. I'll change the roles when ready

**Comment by uxengine** [comment]
Of course, `supporter` is for admin tools, and `admin` is for ... whatever ;)

Removed`supporter` now.

**Pull request merged**